plugins/html-elements-plugin/src/main/java/ru/sbtqa/tag/pagefactory/stepdefs/HtmlSetupStepDefs.java

```diff
@@ -1,14 +1,19 @@
-package ru.sbtqa.tag.pagefactory.html.stepdefs;
+package ru.sbtqa.tag.pagefactory.stepdefs;
 
-import cucumber.api.java.Before;
 import ru.sbtqa.tag.pagefactory.environment.Environment;
 import ru.sbtqa.tag.pagefactory.html.actions.HtmlPageActions;
 import ru.sbtqa.tag.pagefactory.web.drivers.WebDriverService;
 
 public class HtmlSetupStepDefs {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)